app/views/mailers/pipeline_mailer/complete_manager_email.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= render partial: "mailers/email_template",
+locals: {
+  name: @workflow_execution.submitter.first_name,
```

Comment:
This will show `Hello BOT_NAME,` in the email. I think it's fine to just remove name and have the generic `Hello,` in the email.

Reply:
PR screenshots updated. Please see 45f384cd3148e78efb19937acdd8c24dd6692374.